x-pack/plugins/apm/server/index.ts

```diff
@@ -33,6 +36,9 @@ export function mergeConfigs(apmOssConfig: APMOSSConfig, apmConfig: APMXPackConf
     'xpack.apm.ui.maxTraceItems': apmConfig['ui.maxTraceItems'],
     'xpack.apm.ui.transactionGroupBucketSize': apmConfig['ui.transactionGroupBucketSize'],
     'xpack.apm.autocreateApmIndexPattern': apmConfig.autocreateApmIndexPattern,
+    'xpack.apm.serviceMapIndexPattern': apmConfig.serviceMapIndexPattern,
+    'xpack.apm.serviceMapDestinationIndex': apmConfig.serviceMapDestinationIndex,
+    'xpack.apm.serviceMapDestinationPipeline': apmConfig.serviceMapDestinationPipeline,
```

Comment:
What's the purpose of making this configurable?

Reply:
it is to optionally provide a pipeline for the bulk-created docs with connection data. I'm doubtful this is really necessary, so it can safely be removed.